WireAPI/Sources/WireAPI/APIs/FeatureConfigsAPI/FeatureConfigsAPIV1.swift

```diff
@@ -25,19 +25,28 @@ class FeatureConfigsAPIV1: FeatureConfigsAPIV0 {
     }
 
     override func getFeatureConfigs() async throws -> [FeatureConfig] {
-        let request = HTTPRequest(
-            path: "\(pathPrefix)/feature-configs",
-            method: .get
-        )
+        let components = URLComponents(string: resourcePath)
+
+        guard let url = components?.url else {
```

Comment:
This is kind of repeated everywhere (excluding the error type). I think this should be de-duplicated. For example, by introducing a throwing way of getting the URL.

I also think the pattern of having different errors `ConnectionsAPIError.invalidURL` , `FeatureConfigsAPIError.invalidURL` that mean essentially the same thing can be a kind of code smell as it prevents generic handling of such errors.

Reply:
There is a ticket to factor out common api errors and re-use them. As for the repetitive guard let, either we could add a convenience throwing method, or perhaps we can integrate it into the url request builder.

Reply:
done in https://github.com/wireapp/wire-ios/pull/2249/commits/10b51780771639e68c672830917b4e4529c1eb43
We actually had already what we needed in URLRequestBuilder.